src/status_im/chat/models.cljs

```diff
@@ -36,9 +37,9 @@
 
 (defn update-chat
   "Updates chat properties, if chat is not present in db, creates a default new one"
-  [{:keys [db get-stored-chat]} {:keys [chat-id] :as chat}]
+  [{:keys [db get-stored-chat] :as cofx} {:keys [chat-id] :as chat}]
   (let [chat (merge (or (get-stored-chat chat-id)
-                        (create-new-chat db chat-id {}))
+                        (create-new-chat cofx chat-id {}))
```

Comment:
Can we make this cofx explicit in create-new-chat signature as well? In my checkout it doesn't mention cofx

Reply:
it doesn't have to, create-new-chat just needs to be passed a map that contains db and now

Reply:
No I understand this, but if we are clarifying elsewhere in this ns here let's clarify it in signature too. It isn't unlikely this is why db got passed in the first place.